x-pack/plugins/ml/public/application/management/jobs_list/components/jobs_list_page/space_management/space_management.tsx

```diff
@@ -35,9 +35,10 @@ import { getFilters } from './filters';
 interface Props {
   spacesApi?: SpacesPluginStart;
   setCurrentTab: (tabId: MlSavedObjectType) => void;
+  setRefreshJobs: (refresh: (() => void) | null) => void;
```

Comment:
shall we consider a more common naming convention, e.g. `onReload`

Reply:
the one on top can also be `onTabChange`

Reply:
Updated in f55c97f22f14e3996b40610143d0413c7ae1a6df